.github/workflows/publish_hotfix.yaml

```diff
@@ -0,0 +1,76 @@
+name: Publish Temporary ROCK
+
+on:
+  pull_request:
+    types: [ labeled ]
```

Comment:
Does this say we'd push new images on all PRs that are labeled or would this be restricted to authorized contributors?

Reply:
This action will trigger when PRs raised from a branch on `canonical/ceph-containers` (rather than a fork based PR where GH_TOKEN does not have `Package: write` permission) are labelled as hotfix. So can only be triggered by maintainers and above.

Reply:
But L13 below with the `permissions` key would grant write privileges on packages, wouldn't it? How does the restriction to branches work?
Sorry for the many questions

Reply:
@UtkarshBhatthere ping just so that the question above ^^ doesn't get lost